spec/ndx-holographic-stimulation.extensions.yaml

```diff
@@ -22,24 +79,32 @@ groups:
     doc: link to the holographic stimulus pattern
     quantity: '?'
 - neurodata_type_def: HolographicSeries
-  neurodata_type_inc: OptogeneticSeries
+  neurodata_type_inc: TimeSeries
   doc: An extension of OptogeneticSeries to include the holographic representation
     for the stimulus.
+  attributes:
+  - name: unit
+    dtype: text
+    default_value: W
```

Comment:
this should follow core-schema naming:
https://github.com/NeurodataWithoutBorders/nwb-schema/blob/ec0a87979608c75a785d3a42f61e3366846ed3c2/core/nwb.ogen.yaml#L16